docs/upgrading-to-v4.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading to v4
```

Comment:
Also worth mentioning that it's available now in beta, and how to get it?

Reply:
Fair point. Added.